app/experimenter/experiments/models.py

```diff
@@ -22,6 +22,13 @@
 class ExperimentManager(models.Manager):
 
     def get_queryset(self):
+        return (
+            super()
+            .get_queryset()
+            .annotate(latest_change=Max("changes__changed_on"))
+        )
+
+    def get_prefetched(self):
         return (
             super()
```

Comment:
I think if you omit the super() call here, then you'll call the subclassed get_queryset written above, which means you'll get the annotation for free so you can also omit it in get_prefetched?